### PR TITLE
Create Doneness document

### PR DESCRIPTION
**Pull request by LydiaFrance**
Following a tech talk on 13/09/2022, I was requested to add this document to the Handbook.

### CODE REVIEW

File: content/docs/how_we_work/defining-done.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+> Assuming you're starting with a code base that already exists, this document is designed to help you discuss scope of RSE involvement
+> with collaborators and define when a project is done.
+> This means making it easier for your current and future research, accelerating the impact and legacy of the software,
+> expanding communities of practice, and providing a confident codebase for future research.
+> This framework highlights what a maximal 'definition of done' would look like for mutual success to deliver reproducible and
+> credible code for all. It is expected that not all engagements will hit the maximum extents, but rather this is to be negotiated
+> and shaped by the resources available.
+
+>For each project discuss which "Level" in each of the following categories is a MINIMUM aim for defining done:
```
Comment (JimMadge) [inline content/docs/how_we_work/defining-done.md:25]:
```suggestion
This document was produced as part of Accelerating AI in the Arts and Humanities (AAAH) produces outputs of professional quality for the research community.

Assuming you're starting with a code base that already exists, this document is designed to help you discuss scope of RSE involvement
with collaborators and define when a project is done.
This means making it easier for your current and future research, accelerating the impact and legacy of the software,
expanding communities of practice, and providing a confident codebase for future research.
This framework highlights what a maximal 'definition of done' would look like for mutual success to deliver reproducible and
credible code for all. It is expected that not all engagements will hit the maximum extents, but rather this is to be negotiated
and shaped by the resources available.

For each project discuss which "Level" in each of the following categories is a MINIMUM aim for defining done:
```

**Review by JimMadge — COMMENTED** [review]
Thanks @LydiaFrance. Looks good.

The paragraphs probably don't need to be quoted.

Maybe [definition lists](https://www.markdownguide.org/extended-syntax/#definition-lists) would look nice for the levels?

Feel free to merge when you are happy :+1:

File: content/docs/how_we_work/defining-done.md (hunk 1 above)
Comment (LydiaFrance) [reply content/docs/how_we_work/defining-done.md]: Thanks, I made those changes. I agree definitions look better but they didn't play well with the subtitle markup. i've added `&nbsp;` as a hack solution... 😅

**Comment by JimMadge** [comment]
Hi @LydiaFrance are you happy to merge this?

**Pull request merged**